packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayerContainer.spec.js

```diff
@@ -2,49 +2,21 @@ import { shallow } from 'enzyme'
 import React from 'react'
 import InteractionLayerContainer from './InteractionLayerContainer'
 
-const activeStepTasksWithDrawing = [
-  {
-    instruction: 'Draw a point and line',
-    taskKey: 'T0',
-    tools: [
-      { type: 'point' },
-      { type: 'line' }
-    ],
-    type: 'drawing'
-  }
-]
-
-const activeStepTasksWithSingleChoice = [
-  {
-    answers: [{ label: 'yes' }, { label: 'no' }],
-    question: 'Is there a cat?',
-    required: true,
-    taskKey: 'T0',
-    type: 'single'
-  }
-]
-
 describe('Component > InteractionLayerContainer', function () {
   it('should render without crashing', function () {
-    shallow(<InteractionLayerContainer.wrappedComponent />)
+    shallow(
+      <InteractionLayerContainer.wrappedComponent
+        addToStream={() => {}}
+        storeSVG={() => {}}
```

Comment:
These props have been removed and should be replaced with the new props for the container component.

Reply:
Is this update going to happen in this PR or in a follow up?